src/dcat-ap/dcat-dataset.ts

```diff
@@ -3,10 +3,14 @@ import { localeToLang } from './dcat-formatters';
 import * as _ from 'lodash';
 import { isPage } from '@esri/hub-sites';
 import { UserSession } from '@esri/arcgis-rest-auth';
+import { IModel, datasetToContent, DatasetResource, getContentSiteUrls, IHubRequestOptions, getProxyUrl, datasetToItem } from '@esri/hub-common';
 
 // Required fields from the API
 export const defaultRequiredFields = [
   'id',
+  'access', // needed for proxied csv's
```

Comment:
If you don't supply a UserSession, which we don't (for now), all results back are public.

Reply:
As expected! The `isProxiedCSV` helper reads `item.access` as part of it's flow, so I figured it would be cleaner to include it back as part of the api response instead of stabbing in on right before the helper.